apollo-router/src/plugins/telemetry/metrics/apollo/snapshots/apollo_router__plugins__telemetry__metrics__apollo__studio__test__aggregation.snap

```diff
@@ -1465,5 +1466,9 @@ expression: aggregated_metrics
       "subtype": null,
       "licensed_operation_count": 2
     }
-  }
+  },
+  "router_features_enabled": [
+    "apq",
```

Comment:
Shouldn't this be `distributed_apq_cache`?

Reply:
Yes, and the path into the config is:

```
apq: 
  router: 
    cache: 
      redis: 
```